Mathlib/Topology/Algebra/Module/LinearPMap.lean

```diff
@@ -183,4 +183,56 @@ theorem closureHasCore (f : E →ₗ.[R] F) : f.closure.HasCore f.domain := by
   exact domRestrict_apply (hxy.trans hyz)
 #align linear_pmap.closure_has_core LinearPMap.closureHasCore
 
+/-! ### Topological properties of the inverse -/
+
+section Inverse
+
+variable {f : E →ₗ.[R] F} (hf : LinearMap.ker f.toFun = ⊥) (hf' : f.IsClosable)
+
```

Comment:
I think it would be a good idea to move `hf` and `hf'` directly into the lemma statements below. It actually took me a while to figure out how the lemma statements matched their docstrings.